src/Components/NFT/Detail/index.js

```diff
@@ -31,9 +33,25 @@ console.log("here enter")
   };
   let navigate = useNavigate();
 
-  const nft__detail = useSelector((state) => state.nft__detail);
-  console.log(useSelector(s => s.nft__detail));
-  console.log(nft__detail);
+  const nftData = useSelector((state) => state.nft__detail);
+  const nftIdFromUrl = useParams().nftid;
+
+  useEffect(() => {
+    async function getNftDetails() {
+      try {
+        const response = await axios.get(`${API_BASE_URL}/nfts/${nftIdFromUrl}`);
+        dispatch({ type: "nft__detail", payload: mapNftDetails(response.data.data) });
```

Comment:
Before dispatch just check if there is any data

Reply:
what do you mean by this? if the request gets fulfilled, then the response is supposed to have data, isn'it? if not, it will be handled in the catch statement.

Reply:
Jut for extra layer of validation

Reply:
ok np